worker-tasks/sbt

```diff
@@ -0,0 +1,580 @@
+#!/usr/bin/env bash
+#
+# A more capable sbt runner, coincidentally also called sbt.
+# Author: Paul Phillips <paulp@improving.org>
+
```

Comment:
Instead of maintaining our own copy of this file, would it be easier to check it out from [github](https://github.com/paulp/sbt-extras) and copy it into the worker-tasks folder as part of provisioning?